test/extended/util/client.go

```diff
@@ -124,7 +124,15 @@ func NewCLI(project string) *CLI {
 	cli := NewCLIWithoutNamespace(project)
 	cli.withoutNamespace = false
 	// create our own project
-	g.BeforeEach(func() { _ = cli.SetupProject() })
+	g.BeforeEach(func() {
```

Comment:
once the "wait for these things" step is common between the two impls, then I have relatively few fears about drift given the stability of the projectrequest API.

Reply:
Done